### PR TITLE
Moved away from jQuery cookie towards js-cookie

### PR DESCRIPTION
**Pull request by JakeTheSnake3p0**
jQuery cookie is [no longer being supported](https://github.com/carhartl/jquery-cookie/issues/349). At some point devs will move on to using js-cookie for other reasons and it doesn't make sense to have two js cookie solutions in their projects. I haven't done testing on Rails 2 but I can't imagine these changes have done anything to break it. I've tested this on my own Rails apps (4.2+) and it's working.

Mimicks: #35 - not meant to step on any toes, I didn't look at this pull request before making my own :\

### CODE REVIEW

**Comment by pboling** [comment]
👍 I will compare this to the other fork and merge one shortly.  They look nearly identical from a cursory overview.

**Comment by pboling** [comment]
@JakeTheSnake3p0 please rebase on master, fix the conflicts, and I will merge this.

**Comment by JakeTheSnake3p0** [comment]
Not sure if I "rebased" correctly. All the older commits since #e6b9eda still show up in my commit history, but all the files are up to date.

**Comment by pboling** [comment]
It's fine, thanks!

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### 0.3.6 - FEB.06.2017
```
Comment (pboling) [inline CHANGELOG]: This is still wrong, but I can fix it before I release it.  Needs to be a minor version bump, not a patch level version bump.  MAJOR.MINOR.PATCH ;)

**Pull request merged**